server/models/CollectionsModel/index.js

```diff
@@ -0,0 +1,6 @@
+const CollectionsModel = {
+  index: async (db, collection) =>
```

Comment:
`index` is a controller action, I think we should use `all` here.

```suggestion
  all: async (db, collection) =>
```

Reply:
[21d246e](https://github.com/AMP-SCZ/dpdash/pull/344/commits/21d246eb6479a99f1ed04a88b2054072eebd2f2d)